### PR TITLE
Add emulator endpoint example

### PR DESCRIPTION
**Pull request by cberio**
Azure Storage Emulator needs `devstoreaccount1` prefix

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "league/flysystem-azure",
+    "name": "cberio/flysystem-azure",
```
Comment (twistor) [inline composer.json]: This shouldn't be changed.

File: composer.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=5.5.0",
         "league/flysystem": "~1.0",
-        "microsoft/azure-storage": "~0.10.1"
+        "microsoft/azure-storage": "~0.15.0"
```
Comment (twistor) [inline composer.json]: Is this required for the emulator?

**Comment by twistor** [comment]
This seems to be changing a bunch of unrelated things. Can you break this up into multiple pull requests?

**Comment by cberio** [comment]
oops this wasn't meant to be.
close this pr and will make it another.

File: composer.json (hunk 2 above)
Comment (cberio) [reply composer.json]: not for emulator, this is for implement `getUri()`

**Pull request closed without merging**